packages/atlas-service/src/util.ts

```diff
@@ -46,10 +46,21 @@ export function throwIfNetworkTrafficDisabled(
 /**
  * https://www.mongodb.com/docs/atlas/api/atlas-admin-api-ref/#errors
  */
-export function isServerError(
+function isAtlasAPIError(
   err: any
 ): err is { error: number; errorCode: string; detail: string } {
-  return Boolean(err.error && err.errorCode && err.detail);
+  return Boolean(err && err.error && err.errorCode && err.detail);
+}
+
+function isCloudBackendError(err: any): err is {
+  errorCode: string;
+  message: string;
+  version: string;
+  status: string;
+} {
+  return Boolean(
+    err && err.errorCode && err.message && err.version && err.status
+  );
```

Comment:
What is this second one?

Reply:
Sorry I only saw your comment now after I merged. I'm seeing similar code here: https://github.com/10gen/mms/blob/2ee6cb60d064e6eb4d3350d073cba947f1ae26f8/client/packages/components/Access/ServiceAccounts/utils.ts#L223-L232 for example. Don't know if that's related. But I can find results for both message and detail in the codebase.

Hopefully more a question than a review comment :)